quill-cassandra/src/main/scala/io/getquill/context/cassandra/encoding/UdtEncodingMacro.scala

```diff
@@ -0,0 +1,107 @@
+package io.getquill.context.cassandra.encoding
+
+import com.datastax.driver.core.UDTValue
+import io.getquill.context.cassandra.CassandraSessionContext
+import io.getquill.util.Messages._
+import io.getquill.util.OptionalTypecheck
+
+import scala.reflect.macros.blackbox.{ Context => MacroContext }
+
+class UdtEncodingMacro(val c: MacroContext) {
+  import c.universe._
+
+  private val encoding = q"io.getquill.context.cassandra.encoding"
+  private val udtRaw = typeOf[UDTValue]
+
+  private val keyspace = "quill_test"
```

Comment:
Looks like unused variable.

Reply:
@optician Thanks for feedback! This is just wip and not ready yet. I've came with several inew deas and will update PR soon with relevant changes (info at PR description is somehow outdated). If you're interested in this feature, I will notify you once API will be less-more established

Reply:
It would be great! I'm very interested)